Userland/Libraries/LibGUI/Calendar.cpp

```diff
@@ -46,6 +46,14 @@ Calendar::Calendar(Core::DateTime date_time, Mode mode)
         }
     }
 
+    auto default_view = Config::read_string("Calendar"sv, "view"sv, "default_view"sv, "month"sv);
```

Comment:
Config keys, see above.

(Also note that you did capitalize "Month" elsewhere, so this would have been a bug.)